aips/aip-increase-coin-symbol.md

```diff
@@ -0,0 +1,68 @@
+---
+aip: (this is determined by the AIP Manager, leave it empty when drafting)
+title: Increase Coin Symbol Length
+author: <a list of the author's or authors' name(s) and/or username(s), or name(s) and email(s). Details are below.>
+discussions-to (*optional): <a url pointing to the official discussion thread>
+Status: Draft
+last-call-end-date (*optional): <mm/dd/yyyy the last date to leave feedbacks and reviews>
+type: Standard (Framework)
+created: 02/01/2024
+updated (*optional): <mm/dd/yyyy>
+requires (*optional): <AIP number(s)>
+---
+
+# AIP-X - Increase Coin Symbol Byte Length
+
+## Summary
+
+Currently, emojis are 8-bytes, and could be longer.  This AIP proposes to increase the coin symbol byte length to allow
```

Comment:
> emojis are 8-bytes

This is factually incorrect, counterexample (3 bytes): ⏰ https://github.com/econia-labs/emojicoin-dot-fun/blob/902b89a3b8c9606208eda65118bbe9c4d0a58ed5/src/move/emojicoin_dot_fun/sources/hex_codes.move#L23

Reply:
Okay, this is a technicality typo... Where the "max" is 8 bytes correct?

Reply:
There are some that are more than 8 bytes, for example the Scottish flag 🏴󠁧󠁢󠁳󠁣󠁴󠁿 is 28 bytes...:

https://github.com/econia-labs/emojicoin-dot-fun/blob/f2db7f1ffb2107b792f3d94c30b1c2d96d26b66b/src/move/emojicoin_dot_fun/sources/hex_codes.move#L2405

Reply:
What do you propose is the longest then?

Reply:
> the longest

There are emojis that are even longer than 28 bytes, and we had to take some of them out to keep package under size limit: https://github.com/econia-labs/emojicoin-dot-fun/blob/f2db7f1ffb2107b792f3d94c30b1c2d96d26b66b/src/python/move_emojis/utils/data_parser.py#L203-L220

Also pretty much any combination that combines modifiers like skin color or man/woman (especially for the 4-person families) can quickly add up

Reply:
Yeah, I suspect 32-bytes then should be coverage enough of most emojis, plus a decent number of combinations of simple emojis